packages/db/src/query/reports.ts

```diff
@@ -0,0 +1,167 @@
+import { and, desc, eq, gt, lte, or, sql } from "drizzle-orm";
+
+import type { DayStatistics, ReportProps } from "@energyleaf/lib";
+import db from "../";
+import { historyReportConfig, reportConfig, reports, reportsDayStatistics } from "../schema/reports";
+import { user } from "../schema/user";
+
+/**
+ * Update the user report settings data in the database
+ */
+export async function updateReportConfig(
+    data: {
+        receiveMails: boolean;
+        interval: number;
+        time: number;
+    },
+    id: string,
```

Comment:
Es ist nicht klar, welche ID damit gemeint ist, wenn man sich nicht den Code anguckt
```suggestion
    userId: string,
```